core/src/components/toggle/test/basic/toggle.e2e.ts

```diff
@@ -1,7 +1,62 @@
 import { expect } from '@playwright/test';
 import { configs, test } from '@utils/test/playwright';
 
-configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
+configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('toggle: ionChange'), () => {
```

Comment:
It looks like there's no ionBlur test for toggle like there is for Checkbox. I think it may be worth adding just in case, but up to you

Reply:
I forgot to commit them 🤦‍♀️ https://github.com/ionic-team/ionic-framework/pull/30733/commits/1a411f0112cd0ddf203212def0b298163eb7601f

Reply:
Too slow @ShaneK!